src/script/components/DetachedWindow/DetachedWindow.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useEffect, useMemo} from 'react';
+
+import {createPortal} from 'react-dom';
+
+import {StyledApp, THEME_ID} from '@wireapp/react-ui-kit';
+
+import {calculateChildWindowPosition} from 'Util/DOM/caculateChildWindowPosition';
+
+import '../../../style/default.less';
+
+interface DetachedWindowProps {
+  children: React.ReactNode;
+  width?: number;
+  height?: number;
+  onClose: () => void;
+  url?: string;
+  name: string;
+}
+
+export const DetachedWindow = ({children, url = '', name, onClose, width = 600, height = 600}: DetachedWindowProps) => {
+  const newWindow = useMemo(() => {
+    const {top, left} = calculateChildWindowPosition(height, width);
+
+    return window.open(
+      url,
+      name,
+      `
+        width=${width}
+        height=${height},
+        top=${top},
+        left=${left}
+        location=no,
+        menubar=no,
+        resizable=no,
+        status=no,
+        toolbar=no,
+      `,
+    );
+  }, [height, name, url, width]);
+
+  useEffect(() => {
+    if (!newWindow) {
+      return () => {};
+    }
+
+    setTimeout(() => copyStyles(window.document, newWindow.document), 0);
```

Comment:
isn't there a dom event that we could listen to something like dom ready or something along those lines? 
This `setTimeout` seems a little hacky

Reply:
I think the new window being opened on a different domain is an issue here, seems like events like this are not fired in this case.